.github/workflows/ci.yaml

```
@@ -2,10 +2,14 @@
-on: 
-    pull_request:
-        branches:
-        - main
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)